runhouse/main.py

```diff
@@ -590,155 +592,129 @@ def status(
     _print_status(cluster_status, current_cluster)
 
 
-def get_clusters_from_den():
-    get_clusters_params = {"resource_type": "cluster", "folder": rns_client.username}
-    clusters_in_den_resp = rns_client.session.get(
-        f"{rns_client.api_server_url}/resource",
-        params=get_clusters_params,
-        headers=rns_client.request_headers(),
+def _create_output_table(
```

Comment:
can we move some of these private "list" helpers into a separate file and out of main?

Reply:
yeah, agreed. I was thinking to do it as part of the cli and main.py revamp task, wdyt?

Reply:
if its not such a big lift to do now i think that would be good, just to have main be a little cleaner in the meantime as well